process/sync/storageBootstrap/shardStorageBootstrapper_test.go

```diff
@@ -252,3 +259,129 @@ func TestShardStorageBootstrapper_GetCrossNotarizedHeaderNonceShouldWork(t *test
 	assert.Nil(t, err)
 	assert.Equal(t, uint64(2), nonce)
 }
+
+func TestShardStorageBootstrapper_applyCrossNotarizedHeaders(t *testing.T) {
+	t.Parallel()
+
+	t.Run("missing current meta should error", func(t *testing.T) {
+		addedCrossNotarized := make(map[string]data.HeaderHandler)
+		addedTrackedHeaders := make(map[string]data.HeaderHandler)
+
+		bootstrapper, _, currentMeta, crossNotarizedHeaders := setupForApplyCrossNotarizedHeadersTests(addedCrossNotarized, addedTrackedHeaders)
+		_ = bootstrapper.store.GetStorer(dataRetriever.MetaBlockUnit).Remove(currentMeta.hash)
```

Comment:
Optionally add comment: "Let's remove current meta" (and below).

Reply:
done